### PR TITLE
[KEYCLOAK-8129] Describe the newly introduced SSO_HOSTNAME environment variable

### PR DESCRIPTION
**Pull request by iankko**
Also add additional steps to 'Advanced Concepts' section, on how
to configure `fixed` hostname SPI provider

Signed-off-by: Jan Lieskovsky <jlieskov@redhat.com>

### CODE REVIEW

**Comment by iankko** [comment]
@matthewhelmke @stianst @Crumby PTAL (once got a chance)

Thank you, Jan

File: openshift/content/advanced_concepts/advanced_concepts.adoc [hunk 1]
```diff
@@ -247,6 +247,65 @@ and access the RH-SSO administrator console at:
...
+|PORT
+|<all>
+
+|TERMINATION
+|reencrypt
+
+|WILDCARD
+|None
+|===
+. Change its `host:` field to match the hostname specified as the value of the *_SSO_HOSTNAME_* environment variable above.
```
Comment (matthewhelmke) [inline openshift/content/advanced_concepts/advanced_concepts.adoc]: Instead of "Change its..." use "Change the...".

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Only one very small change to request. Nicely done!

**Comment by iankko** [comment]
@matthewhelmke Thanks, Matthew! Fixed and squashed into the previous commit.

**Review by matthewhelmke — DISMISSED** [review]

**Review by stianst — DISMISSED** [review]

**Comment by matthewhelmke** [comment]
@stianst I still approve.
@iankko Can you please resolve the conflicts so we can merge?

**Comment by matthewhelmke** [comment]
@iankko FYI, I looked at the conflict and was not sure which of the options was the correct one; this is why I am asking you to please resolve the conflict.

**Comment by matthewhelmke** [comment]
@stianst @iankko 
I took another pass at this and figured out what I believe is the conflict and correct resolution, but when I pushed, it pushed to my fork's master [1]. How do I merge that in here so we can move on?

[1] https://github.com/matthewhelmke/keycloak-documentation/commit/64d3de465730a0b69644820155a3eca818928433

**Comment by iankko** [comment]
@matthewhelmke @stianst  Fixed the conflict & rebased to current master. Should be again ready for review. Sorry for the delay.

**Review by matthewhelmke — APPROVED** [review]
Thank you, @iankko

**Review by stianst — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Comment by matthewhelmke** [comment]
Thank you, @iankko .

**Pull request merged**